compiler/job.go

```diff
@@ -113,6 +113,9 @@ func (j *Job) Build(readers ...zio.Reader) error {
 	if err != nil {
 		return err
 	}
+	if outputs == nil || len(outputs) == 0 {
+		panic("system error: no outputs")
+	}
```

Comment:
Nits: I don't think we need this check but if you want to add it, I think
1. a better place for it would be kernel/Builder.Build,
2. `if len(outputs) == 0` is sufficient, and
3. it should return an error instead of panicking.

Reply:
It's gone.